backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationService.java

```diff
@@ -272,8 +278,8 @@ public boolean setIdentityVerified(String externalId, boolean verified) {
   }
 
   @Transactional(readOnly = false)
```

Comment:
This is an issue flagged by Sonar as a bug; should make this method `private`

Otherwise ✅ on this